inference/worker/interface.py

```diff
@@ -1,24 +1,36 @@
-from typing import Literal
-
+from typing import List, Optional, Union
 import pydantic
-from oasst_shared.schemas import inference
+from oasst_shared import schemas
 
 
 class GenerateStreamParameters(pydantic.BaseModel):
+    """
+    Parameters for generating a stream.
+    """
+
     max_new_tokens: int = 1024
     do_sample: bool = True
-    top_k: int | None = None
-    top_p: float | None = None
-    typical_p: float | None = None
-    temperature: float | None = None
-    repetition_penalty: float | None = None
-    seed: int | None = None
-    stop: list[str] = []
+    top_k: Optional[int] = None
+    top_p: Optional[float] = None
+    typical_p: Optional[float] = None
+    temperature: Optional[float] = None
+    repetition_penalty: Optional[float] = None
+    seed: Optional[int] = None
+    stop: List[str] = []
```

Comment:
We don't want to use `Optional` or `List` for type hints - in Python 3.10 it is equivalent (and simpler, without imports) to use `| None` and `list`